frontend/services/entities/config_profiles.ts

```diff
@@ -0,0 +1,40 @@
+import { getConfig } from "@testing-library/react";
+import { profile } from "console";
+import sendRequest from "services";
+import endpoints from "utilities/endpoints";
+
+export interface IGetConfigProfileStatusResponse {
+  verified: number;
+  verifying: number;
+  failed: number;
+  pending: number;
+}
+
+export default {
+  getConfigProfileStatus: (
+    uuid: string
+  ): Promise<IGetConfigProfileStatusResponse> => {
+    const { CONFIG_PROFILE_STATUS } = endpoints;
+    // return sendRequest("GET", CONFIG_PROFILE_STATUS(uuid));
+
+    return new Promise((resolve) => {
+      resolve({
+        verified: 0,
+        verifying: 1,
+        failed: 2,
+        pending: 3,
+      });
+    });
```

Comment:
Should we optimistically remove the mocked response here?

Reply:
yes will remove